dockerfile_scripts/build_aws_rocm.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -x
+
+WITH_AWS_TRACE=""
+if [ $# -gt 1 ] ; then
+    if [ "$2" = "1" ] ; then
+	# Tell AWS to build with trace messages enabled
+	WITH_AWS_TRACE="--enable-trace"
+    fi
+    if [ $# -gt 2 ] ; then
+        if [ "$3" = "1" ] ; then
+	    WITH_MPICH=1;
+        fi
+    fi
+fi
+
+OFI=$1
+
+if [ "$OFI" = "1" ]; then
+  apt-get update \
+        && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+                tcsh
+
+  # Install AWS_OFI_NCCL
+  AWS_VER=v1.4.0
+  AWS_VER_NUM=1.4.0
+  AWS_NAME=aws-ofi-rccl-cxi
+  AWS_FILE="${AWS_NAME}-${AWS_VER_NUM}"
+  # cuda install dir likely dependent on BaseOS (i.e. ubuntu 20.02)
+  # in case this changes in the future
+
+  cuda_ver_str=`echo $CUDA_VERSION | awk -F "." '{print $1"."$2}'`
+  CUDA_DIR="/usr/local/cuda-$cuda_ver_str/targets/x86_64-linux"
+  GDRCOPY_HOME="/usr"
+  OFI_INSTALL_DIR=/container/ofi
+  ls $OFI_INSTALL_DIR
+  [ "$WITH_MPICH" = "1" ]&&MPI_INSTALL_DIR=${MPICH_INSTALL_DIR}||MPI_INSTALL_DIR=${OMPI_INSTALL_DIR}
+  AWS_CONFIG_OPTIONS="--prefix ${AWS_PLUGIN_INSTALL_DIR} \
+	  --with-libfabric=${OFI_INSTALL_DIR}            \
+	  --with-rccl=${HOROVOD_NCCL_HOME}               \
```

Comment:
I can't find where HOROVOD_NCCL_HOME is set, similar to https://github.com/determined-ai/environments/blob/main/Dockerfile-default-gpu#L12